src/analytics/config.py

```diff
@@ -11,7 +11,7 @@
 
 TWO_HOURS = 7200  # seconds
 
-BASE_URL = "/api/v1/analytics"
+API_V2_BASE_URL = "/api/v2/analytics"
```

Comment:
@NoahNsimbe , I am just curious as to what happens to the V1 BASE URL after this?
Does this eliminate/expire the V1 BASE URL?

Reply:
this is only used for swagger. Let me add v1 to swagger.